spec/features/item_view_metadata_spec.rb

```diff
@@ -125,6 +125,7 @@
       let(:dro_struct) { instance_double(Cocina::Models::DROStructural, contains: []) }
 
       before do
+        allow(item).to receive(:pid).and_return('fc123ky4567')
```

Comment:
I'd suggest fixing it by providing the pid to the item (with the druid: prefix).